.github/workflows/autobuild.yml

```diff
@@ -251,77 +233,81 @@ jobs:
 
       - name:                       "Post-Build for ${{ matrix.config.config_name }}"
         id:                         step_cmd3_postbuild
-        if:                         ${{ matrix.config.cmd3_postbuild }}
+        if:                         matrix.config.cmd3_postbuild
         run:                        ${{ matrix.config.cmd3_postbuild }} ${{ github.workspace }}
         env:
           jamulus_project_path:       ${{ github.workspace }}
           jamulus_buildversionstring: ${{ needs.create_release.outputs.version_name }}
 
-      # Upload Artifact to Job
```

Comment:
Upload/Release binaries

Reply:
The step in question actually just uploads the artifact to the Actions job. I think the `name:` already covers that. The upload to the release does not happen here, but in a later step.